src/server.cpp

```diff
@@ -1097,16 +1096,16 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
                     for ( i = 0, k = 0; i < iServerFrameSizeSamples; i++, k += 2 )
                     {
                         vecdIntermProcBuf[i] += dGain *
-                            ( static_cast<double> ( vecsData[k] ) + vecsData[k + 1] ) / 2;
+                            ( static_cast<float> ( vecsData[k] ) + vecsData[k + 1] ) / 2;
                     }
                 }
             }
         }
 
-        // convert from double to short with clipping
+        // clip the intermediate sound buffer
```

Comment:
The comment should make clear why we clip here. The previous comment was clear since we clipped from double to short, i.e., the short range.

Reply:
Fixed.